### ISSUE TITLE
Does `python_2_unicode_compatible` break in a current version?

### ISSUE THREAD

**Issue by flowl** [bartTC/django-attachments#68]
Related issue: https://github.com/jjkester/django-auditlog/issues/231

"attachments\models.py", line 9

```
Watching for file changes with StatReloader
Exception in thread django-main-thread:
Traceback (most recent call last):
  File "C:\python-3.7.4.amd64\lib\threading.py", line 926, in _bootstrap_inner
    self.run()
  File "C:\python-3.7.4.amd64\lib\threading.py", line 870, in run
    self._target(*self._args, **self._kwargs)
  File "C:\python-3.7.4.amd64\lib\site-packages\django\utils\autoreload.py", line 53, in wrapper
    fn(*args, **kwargs)
  File "C:\python-3.7.4.amd64\lib\site-packages\django\core\management\commands\runserver.py", line 109, in inner_run
    autoreload.raise_last_exception()
  File "C:\python-3.7.4.amd64\lib\site-packages\django\utils\autoreload.py", line 76, in raise_last_exception
    raise _exception[1]
  File "C:\python-3.7.4.amd64\lib\site-packages\django\core\management\__init__.py", line 357, in execute
    autoreload.check_errors(django.setup)()
  File "C:\python-3.7.4.amd64\lib\site-packages\django\utils\autoreload.py", line 53, in wrapper
    fn(*args, **kwargs)
  File "C:\python-3.7.4.amd64\lib\site-packages\django\__init__.py", line 24, in setup
    apps.populate(settings.INSTALLED_APPS)
  File "C:\python-3.7.4.amd64\lib\site-packages\django\apps\registry.py", line 114, in populate
    app_config.import_models()
 File "C:\python-3.7.4.amd64\lib\site-packages\django\apps\config.py", line 211, in import_models
    self.models_module = import_module(models_module_name)
  File "C:\python-3.7.4.amd64\lib\importlib\__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1006, in _gcd_import
  File "<frozen importlib._bootstrap>", line 983, in _find_and_load
  File "<frozen importlib._bootstrap>", line 967, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 677, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "C:\python-3.7.4.amd64\lib\site-packages\attachments\models.py", line 9, in <module>
    from django.utils.encoding import python_2_unicode_compatible
ImportError: cannot import name 'python_2_unicode_compatible' from 'django.utils.encoding' (C:\python-3.7.4.amd64\lib\site-packages\django\utils\encoding.py)
```

**Comment by bartTC** [comment]
Which Django version do you use? Its tested against 3.x, I'd assume it should be fine.

**Comment by bartTC** [comment]
I think your issue was fixed in version 1.5. Please reopen otherwise.
